src/api/glue-api.js

```diff
@@ -0,0 +1,35 @@
+import axios from 'axios'
+// import qs from 'qs'
+
+const ROOT_URL = 'https://glue-api.zapimoveis.com.br/v2/listings?categoryPage=RESULT'
+const villageHouse = '&unitSubTypes=VILLAGE_HOUSE&unitTypes=HOME&usageTypes=RESIDENTIAL&unitTypesV3=VILLAGE_HOUSE'
+// &business=SALE
+// &listingType=USED
+// &portal=ZAP
+// &text=Casa+de+Vila
+// &addressCountry=
+// &addressState=
+// &addressCity=
+// &addressZone=
+// &addressNeighborhood=
+// &addressStreet=
+// &addressAccounts=
+// &addressType=
+// &addressLocationId=
+// &addressPointLat=
+// &addressPointLon=
+// &addressUrl=
+// &size=24
+// &from=0
+
+export async function getAdvertisers(searchTerm) {
```

Comment:
The name of the function informs that will returns the Advertisers data, right?
But, listings is returned instead.

```javascript
...
    return response.data.search.result.listings
}
...
```

What do you think about rename this function?

Reply:
I was thinking in adverts, ended up writing it wrong and got without meaning.
Gonna do it.